pxr/usd/plugin/usdAbc/alembicReader.cpp

```diff
@@ -973,8 +993,24 @@ _ReaderContext::Open(const std::string& filePath, std::string* errorLog)
         _SetupInstancing(instances, promotable, &usedRootNames);
     }
 
+    // Re-root so the <defaultPrim> is actually the archive!
+    TfToken abcReRoot;
+    if (args) {
+        const auto reRoot = args->find("abcReRoot");
+        if (reRoot != args->end())
+            abcReRoot = TfToken(reRoot->second);
+    }
+
     // Fill rest of the cache.
-    _ReadPrimChildren(*this, root, rootPath, *_pseudoRoot);
+    if (!abcReRoot.IsEmpty()) {
+        SdfPath compPath = rootPath.AppendChild(abcReRoot);
+        auto& xform = _prims[compPath];
+        xform.typeName = UsdAbcPrimTypeNames->Xform;
+        xform.specifier = SdfSpecifierDef;
+        _ReadPrimChildren(*this, root, compPath, xform);
+        _pseudoRoot->children.emplace_back(std::move(abcReRoot));
```

Comment:
Will _ReadPrimChildren() set the layer's _defaultPrim_ metadata to abcReRoot?  If not, it would be really nice to make it so, here, now that we're done.

Reply:
I don't believe that is the current behavior, as _defaultPrim_ is handled later (d332f8d).